def main():
    exception()


def exception():
    print("Checking exception")


main()
